mycroft/client/text/main.py

```diff
@@ -1259,17 +1259,10 @@ def simple_cli():
         sys.exit()
 
 
-# Find the correct log path relative to this script
-scriptPath = os.path.dirname(os.path.realpath(__file__))
-localLogPath = os.path.realpath(scriptPath + "/../../../scripts/logs")
-
-# Monitor relative logs (for Github installs)
-start_log_monitor(localLogPath + "/mycroft-skills.log")
-start_log_monitor(localLogPath + "/mycroft-voice.log")
-
-# Also monitor system logs (for package installs)
+# Monitor system logs
 start_log_monitor("/var/log/mycroft-skills.log")
-start_log_monitor("/var/log/mycroft-speech-client.log")
+start_log_monitor("/var/log/mycroft-voice.log")        # when start-mycroft.sh
+start_log_monitor("/var/log/mycroft-speech-client.log")  # when Debian package
```

Comment:
Should we change all references of `voice` to `speech-client` so that they are consistent, especially since they will now be in the same directory?

Reply:
I thought about doing that.  Made the start-mycroft script a little more complex -- that is the one exception.  This was a little lazy but felt like it was a less error prone change.  Or I might just be justifying my laziness.

Reply:
Looking at the /var/log/mycroft/*.log thought below, that would be a reason to stick with this approach.  I can switch to /var/log//mycroft/voice.log here and still support the package install for now.  Later we can change the packages to use the mycroft subdirectory.  I think that is all safe and definitely does simplify the permission issues.